config/commands/edit_linting.sh

```diff
@@ -46,7 +46,30 @@ edit() {
         return
     fi
 
-    local linter_cmd="flake8 --isolated --select=F821,F822,F831,E111,E112,E113,E999,E902"
+    # Prep a JSX-enabled config file for eslint 9
+    cat <<EOF > eslint.config.mjs
```

Comment:
won't that pollute the git dir that we're currently in? Probably should put the config somewhere else and then specify the path to it...